packages/react-composites/src/composites/CallWithChatComposite/components/DialpadModal.tsx

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import React from 'react';
+/* @conditional-compile-remove(PeoplePaneDropdown) */
+import { useState } from 'react';
+/* @conditional-compile-remove(PeoplePaneDropdown) */
+import { useMemo } from 'react';
+/* @conditional-compile-remove(PeoplePaneDropdown) */
+import { Dialpad, DialpadStyles, _DrawerMenu, _DrawerMenuItemProps, _DrawerSurface } from '@internal/react-components';
+/* @conditional-compile-remove(PeoplePaneDropdown) */
+import { IButtonStyles, IModalStyles, Modal, Stack, useTheme, Text, IconButton, PrimaryButton } from '@fluentui/react';
+/* @conditional-compile-remove(PeoplePaneDropdown) */
+import { Call20Regular } from '@fluentui/react-icons';
+/* @conditional-compile-remove(PeoplePaneDropdown) */
+import { drawerContainerStyles } from '../styles/CallWithChatCompositeStyles';
+
+/** @private */
+export interface DialpadModalStrings {
```

Comment:
See another comment below.

This component shows a modal or a drawer based on whether we're on mobile. As such, perhaps just name it `DialpadContainer`, or better still `PreparedDialpad` (that's what it really is -- it brings together styles, hooks together things for dialpad). I've done something similar for buttons in the control bar.